pkg/xray/policies.go

```diff
@@ -19,6 +19,8 @@ type PolicyRuleCriteria struct {
 	// Security Criteria
 	MinimumSeverity string           `json:"min_severity,omitempty"` // Omitempty is used because the empty field is conflicting with CVSSRange
 	CVSSRange       *PolicyCVSSRange `json:"cvss_range,omitempty"`
+	// Omitempty is used in FixVersionDependant because an empty field throws an error in Xray below 3.44.3
+	FixVersionDependant bool `json:"fix_version_dependant,omitempty"`
```

Comment:
Code alignment? `make fmt`?

Reply:
it's broken by the comment, have to fix it manually

Reply:
That's because the comment line is not at the end of the code line.